build/DependencyVersions.props

```diff
@@ -1,15 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <MicrosoftAspNetCoreAllPackageVersion>2.1.1-rtm-30818</MicrosoftAspNetCoreAllPackageVersion>
+    <MicrosoftAspNetCoreAllPackageVersion>2.1.1</MicrosoftAspNetCoreAllPackageVersion>
     <MicrosoftAspNetCoreAppPackageVersion>$(MicrosoftAspNetCoreAllPackageVersion)</MicrosoftAspNetCoreAppPackageVersion>
-    <MicrosoftAspNetCoreDeveloperCertificatesXPlatPackageVersion>2.1.0</MicrosoftAspNetCoreDeveloperCertificatesXPlatPackageVersion>
-    <MicrosoftAspNetCoreMvcPackageVersion>2.1.1</MicrosoftAspNetCoreMvcPackageVersion>
-    <DotnetEfPackageVersion>2.1.1-rtm-30818</DotnetEfPackageVersion>
-    <DotnetDevCertsPackageVersion>2.1.0</DotnetDevCertsPackageVersion>
-    <DotnetSqlCachePackageVersion>2.1.0</DotnetSqlCachePackageVersion>
-    <DotnetUserSecretsPackageVersion>2.1.0</DotnetUserSecretsPackageVersion>
-    <DotnetWatchPackageVersion>2.1.0</DotnetWatchPackageVersion>
+    <MicrosoftAspNetCoreDeveloperCertificatesXPlatPackageVersion>$(MicrosoftAspNetCoreAllPackageVersion)</MicrosoftAspNetCoreDeveloperCertificatesXPlatPackageVersion>
```

Comment:
I'd recommend keeping these variables separate. In the 2.1.2, the value of MicrosoftAspNetCoreAllPackageVersion  will update to 2.1.2, but MicrosoftAspNetCoreDeveloperCertificatesXPlatPackageVersion will stay at 2.1.1